tutorials/W2D5_Mysteries/W2D5_Tutorial3.ipynb

```diff
@@ -0,0 +1,1331 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "89a00b06-154b-4aaf-8bee-b96a675406b5",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/neuromatch/NeuroAI_Course/blob/main/tutorials/W2D5_Mysteries/student/W2D5_Tutorial1.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a> &nbsp; <a href=\"https://kaggle.com/kernels/welcome?src=https://raw.githubusercontent.com/neuromatch/NeuroAI_Course/main/tutorials/W2D5_Mysteries/student/W2D5_Tutorial1.ipynb\"  target=\"_parent\"><img src=\"https://kaggle.com/static/images/open-in-kaggle.svg\" alt=\"Open in Kaggle\"/></a>"
```

Comment:
these links are pointing to the wrong book. 

`W2D5_Tutorial1` instead of `W2D5_Tutorial3` 

Note: the correct urls will not work until these changes are merged into main.

We will need to make sure that whenever we add a new book, or update naming, we also update these links. Have we added a new book so far?

Reply:
Updated links in the Open in [Colab/Kaggle] buttons. Thanks for spotting that!